bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```
@@ -42,7 +42,7 @@ message.event.triggered = Event triggered: {0}
 message.event.filtered = Event filtered: {0}
 message.coap.init.failed = Unable to start CoIoT: {0}
 message.discovery.disabled = Device is marked as non-discoverable, will be skipped
-message.discovery.protected = Device {0} reported 'Access defined' (missing userid/password or incorrect).
+essage.discovery.protected = Device {0} is protected and reports 'Access denied', check userId/password
```

I assume first character was deleted by mistake?
```suggestion
message.discovery.protected = Device {0} is protected and reports 'Access denied', check userId/password
```